client/jetpack-cloud/components/sidebar/style.scss

```diff
@@ -2,22 +2,6 @@
 // sidebar will implement by default.
 
 .jetpack-cloud-sidebar {
-	// Make the sidebar full-height, minus top and bottom padding
```

Comment:
Did we remove the previously added styles for the new navigation changes here?

Reply:
Actually, I just moved some of the styling to `sidebarv2`'s CSS, which I thought was part of the base style. I left the site selector since I thought this would not be part of the sidebarV2 but a Jetpack-specific component.

I also want to mention that I slightly changed some styling to simplify it. For instance, I set the `box-sizing: border-box` to prevent us from calculating full height with paddings and borders in mind.